analyzers/src/SonarAnalyzer.CFG/ShimLayer/OperationInterfaces.xml

```diff
@@ -827,6 +858,7 @@
         <summary>Referenced property.</summary>
       </Comments>
     </Property>
+    <!--<Property Name="ConstrainedToType" Type="ITypeSymbol?" Override="true"/> Commented out to avoid duplication-->
```

Comment:
Please mark comments by us with "Sonar:" to distinguish them from the Roslyn comments.

Consider adding simple support for "Override" by excluding them in
```cs
                this.IsSkipped = this.Type switch
                {
                    "ArgumentKind" => true,
                    "BranchKind" => true,
                    "CaseKind" => true,
                    "CommonConversion" => true,
                    "ForEachLoopOperationInfo" => true,
                    "IDiscardSymbol" => true,
                    "InstanceReferenceKind" => true,
                    "LoopKind" => true,
                    "PlaceholderKind" => true,
                    _ => !this.IsPublicProperty && !this.IsOverride, // <- IsOverride needs to be initiated in the ctor.
                };
```

If you do so, we need an issue tracking the work to include support for overrides.

Reply:
#8283